src/index.spec.ts

```diff
@@ -7,6 +7,159 @@ import {
   createProduct 
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
-});
+describe('isInteger',()=>{
+  it('should return true if the value is integer',()=>{
+    const expected = true;
+    const value = 1;
+    const result = isInteger(value);
+    expect(result).toBe(expected);
+  })
+  it('should be string',()=>{
+    const expected = false;
+    const value = "1";
+    const result = isInteger(value);
+    expect(result).toBe(expected);
+  })
+})
+
+describe('toLowerCase', ()=>{
+  it('should return deaulft message if the value is an empty string', ()=>{
+    const defaultMessage = 'Please provide a string';
+    const result = toLowerCase("");
+    const expected = defaultMessage;
+    expect(result).toBe(expected);
+  })
+  it('should return a message in lower case', ()=>{
+    const result = toLowerCase("MeSsAgE 2022");
+    const expected = "message 2022"
+    expect(result).toBe(expected);
+  })
+})
```

Comment:
Try to add a blank line after each `it` function. And also, before your assertions start. Like this:
```
describe('toLowerCase', () => {
  it('should return deaulft message if the value is an empty string', () => {
    const defaultMessage = 'Please provide a string';
    const result = toLowerCase("");
    const expected = defaultMessage;
    
    expect(result).toBe(expected);
  })
  
  it('should return a message in lower case', ()=>{
    const result = toLowerCase("MeSsAgE 2022");
    const expected = "message 2022"
    
    expect(result).toBe(expected);
  })
})
```